go.sum

```diff
@@ -43,6 +43,7 @@ github.com/hyperledger/fabric-lib-go v1.0.0 h1:UL1w7c9LvHZUSkIvHTDGklxFv2kTeva1Q
 github.com/hyperledger/fabric-lib-go v1.0.0/go.mod h1:H362nMlunurmHwkYqR5uHL2UDWbQdbfz74n8kbCFsqc=
 github.com/hyperledger/fabric-protos-go v0.0.0-20191121202242-f5500d5e3e85 h1:bNgEcCg5NVRWs/T+VUEfhgh5Olx/N4VB+0+ybW+oSuA=
 github.com/hyperledger/fabric-protos-go v0.0.0-20191121202242-f5500d5e3e85/go.mod h1:xVYTjK4DtZRBxZ2D9aE4y6AbLaPwue2o/criQyQbVD0=
+github.com/hyperledger/fabric-sdk-go/test/integration v0.0.0-20200106161850-8f3d32c9d1a6 h1:uF55ZFL1vtCZFJFMLY9bIVVmAVgrW5YgPOCO3tfuk50=
```

Comment:
shouldn't see test/integration in top-level go.mod / go.sum

perhaps you need to run `go mod tidy`.

Reply:
not sure how this got there - I've removed it